examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImWriteCommand.java

```diff
@@ -0,0 +1,79 @@
+package com.matter.controller.commands.pairing;
+
+import chip.devicecontroller.ChipDeviceController;
+import chip.devicecontroller.GetConnectedDeviceCallbackJni.GetConnectedDeviceCallback;
+import chip.devicecontroller.WriteAttributesCallback;
+import chip.devicecontroller.model.AttributeWriteRequest;
+import chip.devicecontroller.model.ChipAttributePath;
+import com.matter.controller.commands.common.CredentialsIssuer;
+import java.util.ArrayList;
+import java.util.logging.Level;
+
+public final class PairOnNetworkLongImWriteCommand extends PairingCommand
+    implements WriteAttributesCallback, GetConnectedDeviceCallback {
+  private static final int MATTER_PORT = 5540;
+  private long devicePointer;
+
+  public PairOnNetworkLongImWriteCommand(
+      ChipDeviceController controller, CredentialsIssuer credsIssue) {
+    super(
+        controller,
+        "onnetwork-long-im-write",
+        PairingModeType.ON_NETWORK,
+        PairingNetworkType.NONE,
+        credsIssue,
+        DiscoveryFilterType.LONG_DISCRIMINATOR);
+  }
+
+  @Override
+  protected void runCommand() {
+    // boolean true for tlv
+    byte[] booleanTLV = {0x09};
+    AttributeWriteRequest attribute = AttributeWriteRequest.newInstance(0, 0x0028, 16, booleanTLV);
```

Comment:
Consider defining `0x0028` and this `16` as constants, and maybe add a comment for the `0`? I'm assuming these are endpoint, cluster, and attribute ID? So it could look something more like:

```
AttributeWriteRequest attribute = 
  AttributeWriteRequest.newInstance(
    /* endpointId= */ 0,
    CLUSTER_ID_BASIC,
    ATTR_ID_LOCAL_CONFIG_DISABLED,
    booleanTLV
  );
```

Reply:
done, thanks